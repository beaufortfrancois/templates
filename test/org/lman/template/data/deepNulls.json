{
  "one": null,
  "two": {
    "foo": "...",
    "bar": null
  }
}
