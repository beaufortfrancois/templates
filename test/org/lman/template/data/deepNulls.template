{{?one}}one{{/one}}
{{?one.foo}}one.foo{{/one.foo}}
{{?one.bar}}one.bar{{/one.bar}}
{{?one.baz}}one.baz{{/one.baz}}
{{?two}}two{{/two}}
{{?two}}{{?foo}}two inherit?{{/foo}}{{/two}}
{{#two}}{{?foo}}two inherit#{{/foo}}{{/two}}
{{?two.foo}}two.foo{{/two.foo}}
{{?two.bar}}two.bar{{/two.bar}}
{{?two.baz}}two.baz{{/two.baz}}
{{?three}}three{{/three}}
{{?three.foo}}three.foo{{/three.foo}}
{{?three.bar}}three.bar{{/three.bar}}
{{?three.baz}}three.baz{{/three.baz}}
