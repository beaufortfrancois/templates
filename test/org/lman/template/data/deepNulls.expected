two
two inherit?
two inherit#
two.foo
